src/models/Wallet.ts

```diff
@@ -86,12 +133,28 @@ export default class Wallet {
     return new ComposedTransaction(json);
   }
 
+  /**
+   * Submit a transaction asynchronously. Refer to <a href="https://developers.zumo.money/docs/guides/send-transactions#submit-transaction">Send Transactions</a> guide for usage details.
+   *
+   * @param composedTransaction Composed transaction retrieved as a result
+   *                             of one of the compose transaction methods
+   */
   async submitTransaction(composedTransaction: ComposedTransaction) {
     const json = await RNZumoKit.submitTransaction(composedTransaction.json);
 
     return new Transaction(json);
   }
 
+  /**
+   * Compose exchange asynchronously. Refer to <a href="https://developers.zumo.money/docs/guides/make-exchanges#compose-exchange">Make Exchanges</a> guide for usage details.
```

Comment:
I think this is a much better approach, guiding them to a more fleshed out explanation rather than just a little inline comment 🚀